postgresql-examples/query-operation/query_operation.bal

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerinax/postgresql;
+import ballerina/sql;
+
+// The username , password and name of the PostgreSQL database
+configurable string dbUsername = "postgres";
+configurable string dbPassword = "postgres";
+configurable string dbName = "postgres";
+configurable int port = 5432;
+
+// Defines a record to load the query result schema as shown below in the
+// 'getDataWithTypedQuery' function. In this example, all columns of the
+// customer table will be loaded. Therefore, the `Customer` record will be
+// created with all the columns. The column name of the result and the
+// defined field name of the record will be matched case insensitively.
+type Customer record {|
+    int customerId;
+    string lastName;
+    string firstName;
+    int registrationId;
+    float creditLimit;
+    string country;
+|};
+
+public function main() returns error? {
+    // Runs the prerequisite setup for the example.
+    check beforeExample();
+
+    // Initializes the PostgreSQL client.
+    postgresql:Client dbClient = check new (username = dbUsername,
+                password = dbPassword, database = dbName);
+
+    // Select the rows in the database table via the query remote operation.
+    // The result is returned as a stream and the elements of the stream can
+    // be either a record or an error. The name and type of the attributes
+    // within the record from the `resultStream` will be automatically
+    // identified based on the column name and type of the query result.
+    stream<record{}, error> resultStream =
```

Comment:
```suggestion
    stream<Customer, error> resultStream =
```

Reply:
I have done this change in [this query](https://github.com/ballerina-platform/module-ballerinax-postgresql/pull/9/files#diff-33a85d930f482e3c54e26c5ed0f1847ce0107abe259693ed2412c3c0fc553015R86) as this is a sample query for custom return type

Reply:
Better you do it here as well as this is our recommended way